modules/stochastic_tools/include/userobject/PolynomialChaos.h

```diff
@@ -53,17 +53,12 @@ class PolynomialChaos : public SurrogateModel
 
   /// Evaluate mean: \mu = E[u]
   virtual Real computeMean() const { return _coeff[0]; };
-  /// Evaluate standard deviation: \sigma = sqrt(E[u^2] - \mu^2)
+  /// Evaluate variance: \sigma = sqrt(E[(u-\mu)^2])
   virtual Real computeSTD() const;
```

Comment:
Can we change this to computeStandardDeviation(); I know it is long but I prefer long for complete clarity.